example-apps/kitchen-sink-new-router/source/components/angular-directives/hello.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'hello',
+  inputs: ['msg'],
```

Comment:
@igor-ka It is but input and output metadata doesn't show up if use annotations there's already a issue for this https://github.com/rangle/augury/issues/472